include/fem-dem/vans_assemblers.h

```diff
@@ -178,8 +178,8 @@ class VANSAssemblerBDF : public NavierStokesAssemblerBase<dim>
                 -(3.7 - 0.65 * exp(-pow((1.5 - log10(re)), 2) / 2)))
  *  and the momentum exchange coefficient
  *  beta =(0.5 * c_d * M_PI *
-         pow(particle_properties[DEM::PropertiesIndex::dp], 2) / 4) *
-        relative_velocity.norm()
+         pow(particle_properties[DEM::PropertiesIndex<DEM::SolverType::cfd_dem>::dp],
```

Comment:
I am not sure, but could this be "DEM::CFDDEMProperties::PropertiesIndex::dp" instead? (Same for the other modifications in this file)